vcpkg/commands/new.md

```diff
@@ -0,0 +1,136 @@
+---
+title: vcpkg new
+description: Command line reference for the vcpkg new command. Creates a new manifest in the current directory.
+ms.date: 07/05/2023
+---
+# vcpkg new
+
+## Synopsis
+
+```console
+vcpkg new --application
+```
+
+```console
+vcpkg new --name hello --version 1.0
+```
+
+## Description
+
+Writes a new manifest into the current working directory. In `--application` mode, the manifest
+contains no name or version, suitable for use in an end user application that ships in binary form.
+
+Otherwise, `--name` and `--version` are required. `--version-date`, `--version-relaxed`, or
+`--version-string` can force that a particular version format is selected. For more information
+about the versioning formats, see [Versioning reference](../users/versioning.md).
+
+## Options
+
+All vcpkg commands support a set of [common options](common-options.md).
+
+### `--application`
+
+Indicates that the manifest will be used for an end user application shipped in binary forms.

```

Comment:
```suggestion
Creates a manifest without name and version. Suitable for use in end user applications.
```

Reply:
Hmmm I didn't want to describe an option as actually "doing the action" because the command is what does that.

Moreover, I intentionally did not describe 'without name and version' because we want non end-user applications to be filling in name and version, rather than using this as a 'shut up' switch.